src/game/ui/layout.rs

```diff
@@ -0,0 +1,90 @@
+use crate::colors::{BORDER_COLOR_NORMAL, NORMAL_BUTTON_COLOR, WHITE};
+use crate::game::ui::components::*;
+use crate::game::ui::fonts::FONT;
+use bevy::prelude::*;
+
+pub fn spawn_game_ui(mut commands: Commands, asset_server: Res<AssetServer>) {
```

Comment:
We might want to look into non-code ways of defining ui.  Bevy ui layouts are still kinda inflexible and it might be worth looking into loading our layout from a ron file or something.  I'll research if there's any good plugins that solve this well

Reply:
I agree, it's incredibly verbose. I haven't found an alternative I like yet.